x-pack/plugins/security_solution/public/explore/network/components/embeddables/embedded_map.tsx

```diff
@@ -265,28 +176,38 @@ export const EmbeddedMapComponent = ({
     [storage]
   );
 
-  const content = useMemo(() => {
-    if (!storageValue) {
-      return null;
-    }
-    return (
-      <Embeddable>
-        <InPortal node={portalNode}>
-          <MapToolTip />
-        </InPortal>
-
-        <EmbeddableMap maintainRatio={!isIndexError}>
-          {isIndexError ? (
-            <IndexPatternsMissingPrompt data-test-subj="missing-prompt" />
-          ) : embeddable != null ? (
-            <EmbeddablePanel embeddable={embeddable} />
-          ) : (
-            <Loader data-test-subj="loading-panel" overlay size="xl" />
-          )}
-        </EmbeddableMap>
-      </Embeddable>
-    );
-  }, [embeddable, isIndexError, portalNode, storageValue]);
+  const content = !storageValue ? null : (
+    <Embeddable>
+      <InPortal node={portalNode}>
+        <MapToolTip />
+      </InPortal>
+
+      <EmbeddableMap maintainRatio={!isIndexError}>
+        {isIndexError ? (
+          <IndexPatternsMissingPrompt data-test-subj="missing-prompt" />
+        ) : (
+          <services.maps.Map
+            // eslint-disable-next-line react/display-name
+            getTooltipRenderer={() => (tooltipProps: RenderTooltipContentParams) =>
+              <OutPortal node={portalNode} {...tooltipProps} />}
+            mapCenter={{ lon: -1.05469, lat: 15.96133, zoom: 1 }}
+            layerList={layerList}
+            filters={appliedFilters}
+            query={query}
+            onApiAvailable={(api: MapApi) => {
+              // Wire up to app refresh action
+              setQuery({
+                id: 'embeddedMap', // Scope to page type if using map elsewhere
+                inspect: null,
+                loading: false,
+                refetch: () => api.reload(),
+              });
+            }}
+          />
+        )}
+      </EmbeddableMap>
+    </Embeddable>
+  );
```

Comment:
Thanks for cleaning this up for Security Solution @nreese ,

I've tested locally and found there's some styling issue:


<img width="2523" alt="Screenshot 2024-05-08 at 11 20 40" src="https://github.com/elastic/kibana/assets/6295984/a7db72e2-958c-4074-a5c1-848bc298ba98">



I have a suggested solution to fix this:

1. Please rename:
EmbeddableMap  to `EmbeddableMapRatioHolder`

2. Please add:
```
const EmbeddableMapWrapper = styled.div`
  position: relative;
`;

const EmbeddableMap = styled.div`
  height: 100%;
  width: 100%;
  position: absolute;
  top: 0;
`;
```

3. Please update the content:
```suggestion
    const content = !storageValue ? null : (
    <Embeddable>
      <InPortal node={portalNode}>
        <MapToolTip />
      </InPortal>
      <EmbeddableMapWrapper>
        <EmbeddableMapRatioHolder maintainRatio={!isIndexError} />
        {isIndexError ? (
          <IndexPatternsMissingPrompt data-test-subj="missing-prompt" />
        ) : (
          <EmbeddableMap>
            <services.maps.Map
              // eslint-disable-next-line react/display-name
              getTooltipRenderer={() => (tooltipProps: RenderTooltipContentParams) =>
                <OutPortal node={portalNode} {...tooltipProps} />}
              mapCenter={{ lon: -1.05469, lat: 15.96133, zoom: 1 }}
              layerList={layerList}
              filters={appliedFilters}
              query={query}
              onApiAvailable={(api: MapApi) => {
                // Wire up to app refresh action
                setQuery({
                  id: 'embeddedMap', // Scope to page type if using map elsewhere
                  inspect: null,
                  loading: false,
                  refetch: () => api.reload(),
                });
              }}
            />
          </EmbeddableMap>
        )}
      </EmbeddableMapWrapper>
    </Embeddable>
  );

```

expected:


<img width="2518" alt="Screenshot 2024-05-08 at 10 40 47" src="https://github.com/elastic/kibana/assets/6295984/fff61441-2853-4316-8155-62e296ef608b">